api/envoy/extensions/filters/udp/udp_proxy/v3/udp_proxy.proto

```diff
@@ -170,6 +170,19 @@ message UdpProxyConfig {
     bool propagate_response_trailers = 11;
   }
 
+  message UdpAccessLogOptions {
+    // The interval to flush access log. The UDP proxy will flush only one access log when the session
+    // is ended by default. If this field is set, the UDP proxy will flush access log periodically with
+    // the specified interval.
+    // The interval must be at least 1ms.
+    google.protobuf.Duration access_log_flush_interval = 1
```

Comment:
Please document how does this field interact with flush_access_log_on_tunnel_connected.

Reply:
Added, thanks!